common/config/rush/common-versions.json

```diff
@@ -64,11 +64,7 @@
     ],
     "webpack": [
```

Comment:
If there's no differing versions, we should be able to remove this entirely (this field is just to allow different versions of the same dependency across the repo) - but understand if you want to keep that out of this PR in case it causes issues